loops/loop5.go

```diff
@@ -0,0 +1,16 @@
+/* Iterating over a Map */
+
+package main
+
+import "fmt"
+
+func main() {
+	change := map[int]string {
+		1 : "pennies",
+		2 : "diemes",
+		3 : "quarter",
+	}
+	for k, v := range change {
```

Comment:
./loop5.go:13:6: k declared and not used
./loop5.go:13:9: v declared and not used